packages/agreement-process/test/agreementConsumerDocuments.test.ts

```diff
@@ -49,40 +50,51 @@ import {
 
 describe("agreement consumer document", () => {
   describe("get", () => {
-    let agreement1: Agreement;
-
-    beforeEach(async () => {
```

Comment:
It ran, obviously, before each test but only half of those used the agreement it initialized